docs/components/heading-hierarchy.md

```diff
@@ -224,18 +224,7 @@ variations:
       swap them so that date is attached to `.m-meta-header_left` and
       `.a-heading.a-heading__icon` is attached to `.m-meta-header_right`.
     variation_name: Meta header
-usage: >-
-  <h3>Use cases</h3>
-
-
-  Consistent scaling, weights, and capitalization are used to create distinction
-  between heading levels.
-
-
-  <h3>Behavior</h3>
-
-  At screen widths of 600px and below, the Display heading and Headings 1-4 drop
-  in size. Headings 5-6 remain consistent at all screen widths.
+usage: "<h3>Use cases</h3>\n\nConsistent scaling, weights, and capitalization are used to create distinction between heading levels.\n\n<h3> Content guidelines </h3>\n\n* Headings should be sentence case\r\n* Do not include punctuation in headings\n* Keep headings to one line on desktop when possible\n\n<h3>Behavior</h3>\nAt screen widths of 600px and below, the Display heading and Headings 1-4 drop in size. Headings 5-6 remain consistent at all screen widths."
```

Comment:
This renders fine, but let's format it for readability here like it was before the edit (each section on its own line).